packages/flutter/test_fixes/material/tooltip_theme_data.dart.expect

```diff
@@ -0,0 +1,25 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() {
+  // Changes made in https://github.com/flutter/flutter/pull/163314
+  TooltipThemeData tooltipThemeData = TooltipThemeData();
+  tooltipThemeData = TooltipThemeData();
+  tooltipThemeData = TooltipThemeData(constraints: BoxConstraints(minHeight: 15.0));
+  tooltipThemeData = TooltipThemeData(constraints: BoxConstraints(minHeight: 15.0));
+  tooltipThemeData = TooltipThemeData(
+    constraints: BoxConstraints(maxWidth: 20.0),
```

Comment:
Same in this file, I would expect the code after a fix to be valid, not having both constraints and height set.

Reply:
Also fixed.